pkgs/development/python-modules/pymanopt/default.nix

```diff
@@ -8,20 +8,28 @@
   autograd,
   matplotlib,
   pytestCheckHook,
+  setuptools-scm,
 }:
 
 buildPythonPackage rec {
   pname = "pymanopt";
-  version = "2.2.0";
-  format = "setuptools";
+  version = "2.2.0.post20240710";
```

Comment:
This should be `N-unstable-YYYY-MM-DD` if you are pinning a specific rev instead of a git tag. 

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming